st2client/st2client/commands/action.py

```diff
@@ -278,7 +278,48 @@ class ActionDisableCommand(resource.ContentPackResourceDisableCommand):
 
 
 class ActionDeleteCommand(resource.ContentPackResourceDeleteCommand):
-    pass
+    def __init__(self, resource, *args, **kwargs):
+        super(ActionDeleteCommand, self).__init__(resource, *args, **kwargs)
+
+        self.parser.add_argument(
+            "-f",
+            action="store_true",
+            help="Auto yes flag to delete action files from disk.",
+        )
+
+        self.parser.add_argument(
+            "--force",
+            action="store_true",
+            help="Auto yes flag to delete action files from disk.",
+        )
```

Comment:
I noticed another thing here. Please collapse the two arguments into one. `-f` is just an abbreviation for `--force`.  You can follow this for example: https://github.com/StackStorm/st2/blob/master/st2client/st2client/commands/resource.py#L155. The end result should look like the following.
```
vagrant@axon:~/st2$ st2 action delete -h
usage: st2 action delete [-h] [-t TOKEN] [--api-key API_KEY] [-j] [-y] [-f]
                         ref-or-id

Delete an existing action.

positional arguments:
  ref-or-id             Reference or ID of the action.

optional arguments:
  -h, --help            show this help message and exit
  -t TOKEN, --token TOKEN
                        Access token for user authentication. Get
                        ST2_AUTH_TOKEN from the environment variables by
                        default.
  --api-key API_KEY     Api Key for user authentication. Get ST2_API_KEY from
                        the environment variables by default.
  -j, --json            Print output in JSON format.
  -y, --yaml            Print output in YAML format.
  -f, --force           Auto yes flag to delete action files from disk.
```